src/components/molecules/results-list/results-list-item/__tests__/results-list-item.test.js

```diff
@@ -13,11 +13,7 @@ describe('<ResultsListItem />', function () {
 		const nciId = 'NCI-12984';
 		const summary = 'sample summary';
 		const title = 'sample title';
-
-		useStateValue.mockReturnValue([
-			{ resultsItemTitleLink: 'http://sample.com/test-url/v' },
-		]);
-
+		const resultsItemTitleLink = 'http://sample.com/test-url/v?id={{nci_id}}';
```

Comment:
This should probably be changed to /clinicaltrials/{{nci_id}}

Reply:
then expect the following 
expect(container.querySelector('a')).toHaveAttribute(
			'href',
			'/clinicaltrials/NCI-12984'
		);